scripts/genetic_backtester/darwin.js

```diff
@@ -240,6 +241,23 @@ let RangeNeuralActivation = () => {
 };
 
 let strategies = {
+  bollinger: {
+    period_length: RangePeriod(1, 60, 'm'),
+    min_periods: Range(1, 20),
+    markdown_buy_pct: RangeFloat(-1, 5),
+    markup_sell_pct: RangeFloat(-1, 5),
+    order_type: RangeMakerTaker(),
+    sell_stop_pct: Range0(1, 50),
+    buy_stop_pct: Range0(1, 50),
+    profit_stop_enable_pct: Range0(1, 20),
+    profit_stop_pct: Range(1,20),
+
+  // -- strategy
+    bollinger_size: Range(1, 15),
```

Comment:
bollinger_size defaults to 20 in the conf.js. So I'd assume it should be included in the range aswell?

Reply:
thank you,   I missed that in my cleanup before check-in.   I originally had it so low as I knew anything higher in my sample data would result in 0 trades.   but you are correct,  it should be +- atleast 100 % to allow the algorithm to evolve it properly.

Reply:
No problem. I had actually implemented bollinger to darwin locally, but wasn't sure on what would be the right parameter ranges.